pkg/subscraping/sources/scanmalware/scanmalware.go

```
@@ -0,0 +1,284 @@
+		results <- subscraping.Result{Source: s.Name(), Type: subscraping.Error, Error: err}
+		s.errors++
+		session.DiscardHTTPResponse(resp)
+		return false
+	}
+
+	var body ctDNSResponse
+	err = jsoniter.NewDecoder(resp.Body).Decode(&body)
+	if err != nil {
+		results <- subscraping.Result{Source: s.Name(), Type: subscraping.Error, Error: err}
+		s.errors++
+		resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)